[2024-08-01T18:33:46.688+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-01T18:33:46.883+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reddit_data_pipeline.reddit_extraction manual__2024-08-01T18:33:43.260840+00:00 [queued]>
[2024-08-01T18:33:46.910+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reddit_data_pipeline.reddit_extraction manual__2024-08-01T18:33:43.260840+00:00 [queued]>
[2024-08-01T18:33:46.913+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-01T18:33:46.944+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): reddit_extraction> on 2024-08-01 18:33:43.260840+00:00
[2024-08-01T18:33:46.954+0000] {standard_task_runner.py:64} INFO - Started process 53 to run task
[2024-08-01T18:33:46.962+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'reddit_data_pipeline', 'reddit_extraction', 'manual__2024-08-01T18:33:43.260840+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/reddit_dag.py', '--cfg-path', '/tmp/tmpnu8b7cdf']
[2024-08-01T18:33:46.966+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask reddit_extraction
[2024-08-01T18:33:47.061+0000] {task_command.py:426} INFO - Running <TaskInstance: reddit_data_pipeline.reddit_extraction manual__2024-08-01T18:33:43.260840+00:00 [running]> on host 4955d3be83dc
[2024-08-01T18:33:47.299+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Piyush Agarwal' AIRFLOW_CTX_DAG_ID='reddit_data_pipeline' AIRFLOW_CTX_TASK_ID='reddit_extraction' AIRFLOW_CTX_EXECUTION_DATE='2024-08-01T18:33:43.260840+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-01T18:33:43.260840+00:00'
[2024-08-01T18:33:47.302+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-01T18:33:47.362+0000] {logging_mixin.py:188} INFO - Connected to Reddit
[2024-08-01T18:33:48.452+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-01T18:33:48.454+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/pipelines/reddit_pipeline.py", line 11, in reddit_pipeline
    posts_df = pd.Dataframe(posts)
AttributeError: module 'pandas' has no attribute 'Dataframe'
[2024-08-01T18:33:48.490+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=reddit_data_pipeline, task_id=reddit_extraction, run_id=manual__2024-08-01T18:33:43.260840+00:00, execution_date=20240801T183343, start_date=20240801T183346, end_date=20240801T183348
[2024-08-01T18:33:48.524+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 5 for task reddit_extraction (module 'pandas' has no attribute 'Dataframe'; 53)
[2024-08-01T18:33:48.557+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-08-01T18:33:48.689+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-01T18:33:48.701+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
